proposals/0384-alpenglow-migration.md

```diff
@@ -0,0 +1,261 @@
+---
+simd: '0384'
+title: Alpenglow migration
+authors:
+  - Kobi Sliwinski (Anza)
+  - Ashwin Sekar (Anza)
+  - Carl Lin (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2025-10-21
+feature: a1penGLz8Vm2QHYB3JPefBiU4BY3Z6JkW2k3Scw5GWP
+---
+
+## Summary
+
+Migrate to Alpenglow from TowerBFT
+
+## Motivation
+
+Migrating from TowerBFT to Alpenglow consensus requires a safe handoff
+mechanism that doesn't rollback TowerBFT confirmed user transactions.
+
+## Dependencies
+
+This proposal depends on the following accepted proposal:
+
+- **[SIMD-0326]: Alpenglow**
+
+    Requires Alpenglow to be implemented in order to migrate
+
+- **[SIMD-0307]: Add Block Footer**
+
+    Specifies `BlockMarker`, an means of disseminating metadata in a block
+
+
+## New Terminology
+
+Strong optimistic confirmation:
+
+- A block `B` is strong OC if there exists a block `T` such that `B` is the parent
+  of `T` and `slot(B) + 1 = slot(T)` and `T` contains vote transactions for `B` from
+  at least `82%` of stake.
+
+## Detailed Design
+
+### Migration Handoff
+
+1. Pick a migration boundary slot `S` as follows. Let `X` be the rooted slot in
+   which the feature flag is activated. Let the migration slot S be `X + 5000`,
```

Comment:
nit: this sounds like a suggestion, I believe we've made a decision on this value being 5000 already?

Reply:
I believe right now in the code `MIGRATION_SLOT_OFFSET = 32`, we can change this though

Reply:
edit nvm, was looking at tests, it's already correctly set https://github.com/anza-xyz/alpenglow/blob/5e1f448123da6821ea062f85c15d549969185f10/votor-messages/src/migration.rs#L65-L70